largest_prime_factor.py

```diff
@@ -0,0 +1,27 @@
+"""
+The prime factors of 13195 are 5, 7, 13 and 29.
+
+What is the largest prime factor of the number 600851475143? 
+"""
+def is_prime(n):
+	if n <= 2:
+		return False
+	for x in range(2, num1):
+		if x ** 2 > n:
+			break
+		if n % x == 0:
+			return False
+	return True
+
+def factors(num1):
+	output = ()
+	if is_prime == False:
+		for x in range(2, num1):
+			if x % num1 == 0:
+				output += x
```

Comment:
you defined output as an empty tuple, not sure this line would work

Reply:
this is where the code would break and where i ended up watching a video explaining how their code would run